<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dyner</class>
 <widget class="QMainWindow" name="Dyner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dyner</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/img/img/icon.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: #ffffff;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 243, 243);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="menuButton">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 10px;
background-color: rgb(0, 160, 255);
color : white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 160, 255,1), stop:1 rgba(191, 110, 251, 1));</string>
           </property>
           <property name="text">
            <string>Menu</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="menuList">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(243, 243, 243);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="parentButtonHome">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 10px;
color : white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 160, 255,1), stop:1 rgba(191, 110, 251, 1));</string>
             </property>
             <property name="text">
              <string>Home</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="parentButtonOrder">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 10px;
color : white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 160, 255,1), stop:1 rgba(191, 110, 251, 1));</string>
             </property>
             <property name="text">
              <string>ORDER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="parentButtonBillRequest">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 10px;
color : white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 160, 255,1), stop:1 rgba(191, 110, 251, 1));</string>
             </property>
             <property name="text">
              <string>Bill Request</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="parentButtonBillHistory">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 10px;
color : white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 160, 255,1), stop:1 rgba(191, 110, 251, 1));</string>
             </property>
             <property name="text">
              <string>Bill History</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="parentButtonAdmin">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 10px;
color : white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 160, 255,1), stop:1 rgba(191, 110, 251, 1));</string>
             </property>
             <property name="text">
              <string>Admin</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="horizontalFrame">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(243, 243, 243);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="windowContainer"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
